pkg/webhooks/webhooks.go

```diff
@@ -60,6 +60,7 @@ func Initialize(injectCloudProvider func(context.Context, cloudprovider.Options)
 		ServiceName: opts.KarpenterService,
 		SecretName:  fmt.Sprintf("%s-cert", opts.KarpenterService),
 	})
+	ctx = sharedmain.WithHADisabled(ctx) // Disable leader election for webhook
```

Comment:
Can you either add a TODO here or adjust https://github.com/aws/karpenter/issues/2562 to reflect the need for more work after this?

Reply:
I'll add a TODO, makes sense

Reply:
Relevant details were added to the issue as well for the future